--- origsrc/tumbler-0.1.21/plugins/ffmpeg-thumbnailer/Makefile.am	2011-02-12 13:46:10.000000000 -0600
+++ src/tumbler-0.1.21/plugins/ffmpeg-thumbnailer/Makefile.am	2011-02-27 18:07:35.451230300 -0600
@@ -47,6 +47,7 @@ tumbler_ffmpeg_thumbnailer_la_LDFLAGS =
 	-avoid-version							\
 	-export-dynamic							\
 	-module								\
+	-no-undefined							\
 	$(PLATFORM_LDFLAGS)
 
 tumbler_ffmpeg_thumbnailer_la_LIBADD =					\
--- origsrc/tumbler-0.1.21/plugins/font-thumbnailer/Makefile.am	2011-02-12 13:46:10.000000000 -0600
+++ src/tumbler-0.1.21/plugins/font-thumbnailer/Makefile.am	2011-02-27 18:07:40.318438800 -0600
@@ -47,6 +47,7 @@ tumbler_font_thumbnailer_la_LDFLAGS =
 	-avoid-version							\
 	-export-dynamic							\
 	-module								\
+	-no-undefined							\
 	$(PLATFORM_LDFLAGS)
 
 tumbler_font_thumbnailer_la_LIBADD =					\
--- origsrc/tumbler-0.1.22/plugins/gst-thumbnailer/Makefile.am	2011-08-06 08:51:46.000000000 -0500
+++ src/tumbler-0.1.22/plugins/gst-thumbnailer/Makefile.am	2011-10-26 00:00:53.782033600 -0500
@@ -49,6 +49,7 @@ tumbler_gst_thumbnailer_la_LDFLAGS =
 	-avoid-version							\
 	-export-dynamic							\
 	-module								\
+	-no-undefined							\
 	$(PLATFORM_LDFLAGS)
 
 tumbler_gst_thumbnailer_la_LIBADD =					\
--- origsrc/tumbler-0.1.21/plugins/jpeg-thumbnailer/Makefile.am	2011-02-12 13:46:10.000000000 -0600
+++ src/tumbler-0.1.21/plugins/jpeg-thumbnailer/Makefile.am	2011-02-27 18:07:44.811246600 -0600
@@ -47,6 +47,7 @@ tumbler_jpeg_thumbnailer_la_LDFLAGS =
 	-avoid-version							\
 	-export-dynamic							\
 	-module								\
+	-no-undefined							\
 	$(PLATFORM_LDFLAGS)
 
 tumbler_jpeg_thumbnailer_la_LIBADD =					\
--- origsrc/tumbler-0.1.21/plugins/pixbuf-thumbnailer/Makefile.am	2011-02-12 13:46:10.000000000 -0600
+++ src/tumbler-0.1.21/plugins/pixbuf-thumbnailer/Makefile.am	2011-02-27 18:07:49.694055100 -0600
@@ -46,6 +46,7 @@ tumbler_pixbuf_thumbnailer_la_LDFLAGS =
 	-avoid-version							\
 	-export-dynamic							\
 	-module								\
+	-no-undefined							\
 	$(PLATFORM_LDFLAGS)
 
 tumbler_pixbuf_thumbnailer_la_LIBADD =					\
--- origsrc/tumbler-0.1.21/plugins/poppler-thumbnailer/Makefile.am	2011-02-12 13:46:10.000000000 -0600
+++ src/tumbler-0.1.21/plugins/poppler-thumbnailer/Makefile.am	2011-02-27 18:07:52.860860600 -0600
@@ -47,6 +47,7 @@ tumbler_poppler_thumbnailer_la_LDFLAGS =
 	-avoid-version							\
 	-export-dynamic							\
 	-module								\
+	-no-undefined							\
 	$(PLATFORM_LDFLAGS)
 
 tumbler_poppler_thumbnailer_la_LIBADD =					\
--- origsrc/tumbler-0.1.21/plugins/xdg-cache/Makefile.am	2011-02-12 13:46:10.000000000 -0600
+++ src/tumbler-0.1.21/plugins/xdg-cache/Makefile.am	2011-02-27 19:40:05.208102800 -0600
@@ -47,6 +47,7 @@ tumbler_xdg_cache_la_LDFLAGS =						\
 	-avoid-version							\
 	-export-dynamic							\
 	-module								\
+	-no-undefined							\
 	$(PLATFORM_LDFLAGS)
 
 tumbler_xdg_cache_la_LIBADD =						\
@@ -63,6 +64,6 @@ tumbler_xdg_cache_la_DEPENDENCIES =
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(tumbler_plugindir)
 	-( cd $(DESTDIR)$(tumbler_plugindir) ; \
-		test -x tumbler-xdg-cache.so || \
-		ln -sf tumbler-xdg-cache.so tumbler-cache-plugin.so )
+		test -x tumbler-xdg-cache.dll || \
+		ln -sf tumbler-xdg-cache.dll tumbler-cache-plugin.dll )
 endif
